### DISCUSSION TITLE
Template for content type in ContentPicker

### DISCUSSION THREAD

**Discussion by matiaskusack** [OrchardCMS/OrchardCore#15967] [Q&A] [answered]
Hi, I'm having trouble finding the right name for a template.
This is my Hierarchy:

SomeType -> SomePart -> ContentPickerField(OtherType)

The thing is that I can't change the display in the multiselect when I'm creating a new SomeType content item.

I've tried multiple combinations but I'm not getting it yet.
¿Any ideas?

**Comment by sebastienros** [comment]
Are you trying to change how the related items are displayed on the front-end?
Is the issue that you can change it globally but not for each 'SomeType' ?

**Comment by matiaskusack** [comment]
Hello, thanks for answering.

Well, I'm trying to change how SomeType is rendered in the Admin.
Specifically changing how the ContentPicker multiselect shows SomeType.

It's actually rendering content type's title (SomeType has a TitlePart). It also has a numeric field, so I'd like to define a template to control it's rendering.

**Comment by sebastienros** [comment, marked as answer]
You can create a custom Editor for any field or part. There is no generic solution right now, but if you create one it could be specific to your needs, or based on a Liquid template that would be customizable. 

But if you just need to change the Display Text of a type, there is a part to do that, that accepts some templating. It will change it for all usages of the DisplayText though.
